arbos/arbosState/initialize.go

```diff
@@ -51,21 +55,50 @@ func MakeGenesisBlock(parentHash common.Hash, blockNumber uint64, timestamp uint
 	return types.NewBlock(head, nil, nil, nil, trie.NewStackTrie(nil))
 }
 
-func InitializeArbosInDatabase(db ethdb.Database, initData statetransfer.InitDataReader, chainConfig *params.ChainConfig, initMessage *arbostypes.ParsedInitMessage, timestamp uint64, accountsPerSync uint) (common.Hash, error) {
-	stateDatabase := state.NewDatabase(db)
+func TriedbConfig(c *core.CacheConfig) *trie.Config {
```

Comment:
Created a PR on go-ethereum for that: https://github.com/OffchainLabs/go-ethereum/pull/330

Once go-ethereum PR is approved I update it here.
Or I create a new PR later to update it, in case the current PR moves forward faster :)

Reply:
Done.